### DISCUSSION TITLE
Web App is building on (Raspberry Pi) Arm64, but docker image is not

### DISCUSSION THREAD

**Discussion by minozheros** [standardnotes/forum#2686] [Suggestions]
Hello, 
the local web app is building and working but the docker image is not being built properly.. would it be possible to adjust it in a way that a arm64 docker image could be build

Regards Klaus

**Comment by JaspalSuri** [comment]
Hi @minozheros, it's currently not on our roadmap, but I will update this thread if we're able to provide support for arm64 images in the future.
